src/main/java/org/opensearch/sdk/rest/BaseExtensionRestHandler.java

```diff
@@ -206,6 +206,16 @@ protected final String unrecognized(
         return BaseRestHandler.unrecognizedStrings(request, invalids, candidates, detail);
     }
 
+    /**
+    * Creates a new plain text response with OK status and empty JSON content
+    *
+    * @param request the REST request being responded to
+    * @return ExtensionRestResponse with OK status response
+    */
+    protected ExtensionRestResponse okNoContent(RestRequest request) {
+        return new ExtensionRestResponse(request, RestStatus.OK, "application/json; charset=UTF-8", "{}");
```

Comment:
Can we make that JSON content string a constant (or two constants, one for the type and one for the charset)?  Either define our own or see if it's already included as a constant in an existing dependency.

For example we use Apache http core so [this constant](https://javadoc.io/static/org.apache.httpcomponents.core5/httpcore5/5.0.2/org/apache/hc/core5/http/ContentType.html#APPLICATION_JSON) (v5) [(and this for 1.x backport)](https://www.javadoc.io/static/org.apache.httpcomponents/httpcore/4.4.6/org/apache/http/entity/ContentType.html#APPLICATION_JSON) is available.  

Also, we might not even need the charset as [JSON is UTF-8 by default](https://stackoverflow.com/a/9254967/1161484).

Reply:
Looking at the Base Extension Rest Handler again, I see we already have a `createJSONResponse()` method:
```java
    protected ExtensionRestResponse createJsonResponse(RestRequest request, RestStatus status, String fieldName, String responseStr) {
        try {
            return new ExtensionRestResponse(
                request,
                status,
                JsonXContent.contentBuilder().startObject().field(fieldName, responseStr).endObject()
            );
        } catch (IOException e) {
            // This Should Never Happen (TM)
            // If we messed up JSON code above, just send plain text
            return new ExtensionRestResponse(request, status, fieldName + ": " + responseStr);
        }
    }
```
May want to just name your method `createEmptyJsonResponse` and allow the user to pass the OK param.